src/equations/ideal_glm_mhd_2d.jl

```diff
@@ -210,25 +210,26 @@ end
   v1 = rho_v1 / rho
   v2 = rho_v2 / rho
   v3 = rho_v3 / rho
-  kin_en = 0.5 * rho * (v1^2 + v2^2 + v3^2)
-  mag_en = 0.5*(B1^2 + B2^2 + B3^2)
-  p = (equations.gamma - 1) * (rho_e - kin_en - mag_en - 0.5*psi^2)
+  kin_en = 0.5 * (rho_v1 * v1 + rho_v2 * v2 + rho_v3 * v3)
+  mag_en = 0.5 * (B1 * B1 + B2 * B2 + B3 * B3)
+  p_over_gamma_minus_one = (rho_e - kin_en - mag_en - 0.5 * psi^2)
```

Comment:
would it make it (slightly) faster to use `0.5 * psi * psi` or is the power okay in this case?

Reply:
The power is okay here since there are three multiplications and we can definitely not use FMAs here